lms/envs/aws.py

```diff
@@ -1099,6 +1099,39 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Following are the changes made on edx by caliper_tracking app.
+
+# 1. EVENT_TRACKING_BACKENDS settings:
+#      It is responsible for logging events of 'eventtracking' app, our
+#      'caliper_tracking' app appends its custom processor (named
+#      'caliper_tracking.processor.CaliperProcessor') in the pipeline of
+#      'processors' present in 'tracking_logs' of 'EVENT_TRACKING_BACKENDS'.
+#
+# 2. TRACKING_BACKENDS settings:
+#      Logs generated by 'track' app use 'TRACKING_BACKENDS' settings.
+#      'caliper_tracking' replaces its processor with its own custom backend
+#      (named 'CALIPER_TRACKING_BACKENDS') which eventually redirects the logs
```

Comment:
It is not named `CALIPER_TRACKING_BACKEND`. Please use the correct name.

Reply:
done